omnibus/package-scripts/agent/posttrans

```diff
@@ -55,9 +55,15 @@ else
     echo "[ WARNING ]\tCannot detect a supported init system. The datadog-agent package only provides service files for systemd and upstart."
 fi
 
-# SELinux policy not shipped on SUSE for now
-# FIXME: Refactor warning messages
-if [ "$DISTRIBUTION_FAMILY" != "SUSE" ]; then
+INSTALL_SELINUX_POLICY="false"
+if [ "$DISTRIBUTION" = "CentOS" ] || [ "$DISTRIBUTION" = "RedHat" ]; then
+    OS_RELEASE_VERSION=$(grep VERSION_ID /etc/os-release | cut -d = -f 2 | tr -d '"')
```

Comment:
Do we get the underlying redhat version when this runs on fedora ?

Reply:
No, this drops Fedora from the list of OSes where we install the policy. To support Fedora we would need a list of versions where the policy is needed (since we can't just get the equivalent RHEL version like you mention).

I tested on several Fedora versions and the system-probe was working without the policy, so that's why I dropped it. It could happen that different VM images for the same versions of Fedora give different results, though (we have seen this happen with CentOS for example), so if we know of a Fedora versions that didn't work without the policy let me know and I will test it and see if we should install the policy.

Reply:
And to answer your other question about how to fix it on machines with the policy installed when it shouldn't... we could try to detect this in the upgrade path, but I prefer to avoid that complexity. If the number of affected customers justifies it we can still do it, otherwise we have the manual steps in the original issue.